easybuild/easyblocks/l/llvm.py

```diff
@@ -566,6 +592,63 @@ def _set_gcc_prefix(self):
             self.gcc_prefix = gcc_prefix
         self.log.debug("Using %s as the gcc install location", self.gcc_prefix)
 
+    def _set_dynamic_linker(self):
+        """Set the dynamic linker for the build if not the default one."""
+        if self.sysroot:
+            linkers = glob.glob(os.path.join(self.sysroot, '**', 'ld-*.so*'))
+            for linker in linkers:
+                if os.path.isfile(linker) and not os.path.islink(linker):
+                    self.log.info("Using linker %s from sysroot", linker)
+                    self.dynamic_linker = linker
+                    break
+            else:
+                msg = f"No linker found in sysroot {self.sysroot}, using default linker"
+                trace_msg(msg)
+                self.log.warning(msg)
+
+    def _update_ignore_patterns(self):
```

Comment:
@Crivella Managed to refactor this incorrectly, this needs to be:
```suggestion
    def _update_test_ignore_patterns(self):
```